README.md

```diff
@@ -1,5 +1,13 @@
 # Jaeger OpenShift Templates
 
+## Consider using the Jaeger Operator!
```

Comment:
I would be more strict and say it's deprecated in favor of operator.

For a simple installation, we can point users to kubernetes templates which are compatible with OpenShift but not the other way around.

Reply:
Updated